src/java/org/apache/cassandra/db/MutationVerbHandler.java

```diff
@@ -59,6 +64,11 @@ public void doVerb(Message<Mutation> message)
 
         try
         {
+            // Initialize the sensor and set ExecutorLocals
+            RequestSensors sensors = new RequestSensors();
+            ExecutorLocals locals = ExecutorLocals.create(sensors);
+            ExecutorLocals.set(locals);
+
             message.payload.applyFuture(WriteOptions.DEFAULT).thenAccept(o -> respond(message, respondToAddress)).exceptionally(wto -> {
```

Comment:
Yes, but not required for https://github.com/riptano/cndb/issues/8463 probably a separate issue/PR?

Reply:
True, not required, but once we laid down the foundations in the other issue, I think it would now be wasteful to basically create two different issues every time we add new sensors, what do you think?

Reply:
> what do you think

IIRC from the dial tone meeting before the last, we shifted the focus from rate limiting to testing the UCU metrics themselves (and start thinking about billing) - if this is indeed the desired outcome then adding to coordinator doesn't fit. Also generally I prefer smaller PRs from pure logistics stand point (which I know is not a popular opinion)

Having said that, I'm fine adding coordinator changes here bc there doesn't seem to be a consistent mental model on how to define tasks boundaries divide tasks (at least I don't know) at this point of the project.